3001-4000/LIT3311Prosph.xml

```diff
@@ -30,6 +30,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <citedRange unit="item">5219</citedRange>
                </bibl>
             </listBibl>
+         <listWit>
+            <witness corresp="BNFabb18" xml:id="A"/>
```

Comment:
cp. the remarks given here https://github.com/BetaMasaheft/Works/pull/1310/files same applies